iphone/TitaniumKit/TitaniumKit/Sources/API/TiExceptionHandler.m

```diff
@@ -94,45 +94,15 @@ - (void)reportScriptError:(JSValue *)error inJSContext:(JSContext *)context
 
 - (void)showScriptError:(TiScriptError *)error
 {
-  NSArray<NSString *> *exceptionStackTrace = [error valueForKey:@"nativeStack"];
-  if (exceptionStackTrace == nil) {
-    exceptionStackTrace = [NSThread callStackSymbols];
-  }
-
   NSMutableDictionary *errorDict = [error.dictionaryValue mutableCopy];
   [errorDict setObject:[NSNumber numberWithLong:error.column] forKey:@"column"];
   [errorDict setObject:[NSNumber numberWithLong:error.lineNo] forKey:@"line"];
-  if (exceptionStackTrace == nil) {
-    [[TiApp app] showModalError:[error description]];
-  } else {
-    NSMutableArray<NSString *> *formattedStackTrace = [[[NSMutableArray alloc] init] autorelease];
-    NSUInteger exceptionStackTraceLength = [exceptionStackTrace count];
-
-    // re-size stack trace and format results. Starting at index = 1 to not include showScriptError call
-    for (NSInteger i = 1; i < (exceptionStackTraceLength >= 20 ? 20 : exceptionStackTraceLength); i++) {
-      NSString *line = [self removeWhitespace:[exceptionStackTrace objectAtIndex:i]];
-
-      // remove stack index
-      line = [line substringFromIndex:[line rangeOfString:@" "].location + 1];
-
-      [formattedStackTrace addObject:line];
-    }
-    NSString *stackTrace = [formattedStackTrace componentsJoinedByString:@"\n"];
-    [errorDict setObject:stackTrace forKey:@"nativeStack"];
-
-    [[TiApp app] showModalError:[NSString stringWithFormat:@"%@\n\n%@", [error description], stackTrace]];
-  }
+  NSString *stackTrace = [error.formattedNativeStack componentsJoinedByString:@"\n"];
```

Comment:
`formattedNativeStack` is declared as a function and it is getting used as property . Though it can be used, I think it's better to call it as function to have more readability. 
```suggestion
  NSString *stackTrace = [[error formattedNativeStack] componentsJoinedByString:@"\n"];
```